smugline.py

```diff
@@ -89,7 +95,14 @@ def get_filter(self, media_type='images'):
             return ALL_FILTER
 
     def upload_file(self, album, image):
-        self.smugmug.images_upload(AlbumID=album['id'], **image)
+        retries = 5
+        while retries:
+            try:
+                self.smugmug.images_upload(AlbumID=album['id'], **image)
+                return
+            except HTTPError:
+                print("RETRY ", image)
```

Comment:
for consistency, I'll use lowercase "retry " (same as "skipping" / "deleting" etc)